src/styles/globals.css

```diff
@@ -8,7 +8,7 @@ body {
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
     Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
-  background-color: #333333;
+  /* background-color: #333333; */
```

Comment:
Let us remove this commented-out line.

Reply:
Fixed via [71e1add](https://github.com/reactdeveloperske/reactdevske-website/pull/105/commits/71e1addb2a997285667ba7bfdb6c828581b8c852)